x-pack/legacy/plugins/uptime/server/lib/adapters/monitors/__tests__/combine_range_with_filters.test.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { combineRangeWithFilters } from '../elasticsearch_monitors_adapter';
+
+describe('combineRangeWithFilters', () => {
+  it('combines filters that have no filter clause', () => {
+    expect(
+      combineRangeWithFilters('now-15m', 'now', {
+        bool: { should: [{ match: { 'url.port': 80 } }], minimum_should_match: 1 },
+      })
+    ).toMatchSnapshot();
```

Comment:
I'm OK with this as is, but in the future I think it'd be better not to use snapshots here, but rather to manually include the expected values in the test. It keeps more things in one spot.

Snapshots work best when a large complex datastructure needs programmatic updating. This structure is small, and could be typed out by hand.

I don't think we need to delay this PR, and I'd say this is more of a stylistic call, so I'm fine merging this portion as is.

Reply:
I removed the snapshots and put the literals in expect eqls in https://github.com/elastic/kibana/pull/50837/commits/df8e18fecee74b242be1152e6cfd8e327b3c8eab.